src/components/Shared/Buttons.js

```diff
@@ -8,6 +8,22 @@ export const getButtons = platformLabels => [
     image: 'clear.png',
     main: true
   },
+  {
+    key: 'ClearGrid',
+    label: platformLabels.clearGrid,
+    condition: 'isClearedGrid',
+    color: '#f44336',
+    image: 'clear.png',
+    main: false
+  },
+  {
+    key: 'ClearHG', //HeaderGrid
```

Comment:
remove comment

Reply:
kept on purpose to clarify the name and usage

Reply:
then name it clearHeaderGrid